Makefile

```diff
@@ -0,0 +1,29 @@
+all: build check
+
+check: fmt vet lint ## Check your code
+
+revendor:
+	rm glide.lock
+	glide install --strip-vendor
+	glide-vc --use-lock-file --no-tests --only-code
+
+terraform:
+	hack/get-terraform.sh
+
+build: ## build installer binary
+	hack/build.sh
+
+#test: # Run unit test
+#	go test -race -cover ./cmd/...
+
+lint: ## Go lint your code
+	hack/go-lint.sh -min_confidence 0.3 $(go list -f '{{ .ImportPath }}' ./...)
+
+fmt: ## Go fmt your code
+	hack/go-fmt.sh
```

Comment:
You need a trailing `.` (see [here][1]).

[1]: https://github.com/openshift/installer/blame/7f8f39703b99af3d68582a1503884b2498bb4e5d/CONTRIBUTING.md#L39